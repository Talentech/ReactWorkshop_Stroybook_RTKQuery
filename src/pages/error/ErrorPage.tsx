import React from "react";

const ErrorPage = (): JSX.Element => <p>Something went wrong</p>;

export default ErrorPage;
