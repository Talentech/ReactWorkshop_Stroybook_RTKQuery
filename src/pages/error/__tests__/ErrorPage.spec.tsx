import React from "react";
import renderer from "react-test-renderer";
import ErrorPage from "@pages/error/ErrorPage";

describe("<ErrorPage /> component test", () => {
  it("Should properly render ErrorPage component", () => {
    const component = renderer.create(<ErrorPage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
