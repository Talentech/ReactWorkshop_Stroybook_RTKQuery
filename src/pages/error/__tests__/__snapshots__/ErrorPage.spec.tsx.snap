// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> component test Should properly render ErrorPage component 1`] = `
<p>
  Something went wrong
</p>
`;
